isso/wsgi.py

```diff
@@ -144,8 +145,8 @@ def add_cors_headers(status, headers, exc_info=None):
             return start_response(status, headers.to_list(), exc_info)
 
         if environ.get("REQUEST_METHOD") == "OPTIONS":
-            add_cors_headers("200 Ok", [("Content-Type", "text/plain")])
-            return []
+            response = Response()
+            return response(environ, add_cors_headers)
```

Comment:
I don't see why this needs to be changed - the issue is the first argument to werkzeug.datastructures.Headers.